src/components/concept/_ip_addresses.mdx

```diff
@@ -1,4 +1,4 @@
-If you are behind a firewall, are using a virtual private network (VPN), or have network restrictions regarding access, you may need to add the following IP addresses to the **ingress** allowlist and **egress** allowlist:
+If you are behind a firewall, are using a virtual private network (VPN), or have network restrictions regarding access, you may need to add the following IP addresses to the **ingress** allowlist and **egress** allowlist. For most integrations, communication will occur over TCP port 443:
```

Comment:
```suggestion
If you are behind a firewall, are using a virtual private network (VPN), or have network restrictions regarding access, you may need to add the following IP addresses to the **ingress** allowlist and **egress** allowlist. Communication to and from these IP addresses occurs over TCP port 443 (HTTPS).
```

Reply:
I think we can be this definitive here, but let's get an SME review to make sure.

In the context of the setup doc (which is one of the two places this is used), there's a following section about Broker that explains it more specifically, which I think is sufficiently clear.